related-files-recipe.el

```diff
@@ -36,7 +36,7 @@
 
 ;;; Overrides of Public Methods
 
-(cl-defmethod related-files-apply ((jumper (head recipe)) place)
```

Comment:
this is great! This kind of jumpers only apply to filename places so this change makes perfect sense.

Reply:
Nice, thanks :)